cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -55,6 +56,18 @@ def adata(self, adata: anndata.AnnData):
         self.reset()
         self._adata = adata
 
+    def _set_is_spatial(self) -> bool:
```

Comment:
change this to be a check if the dataset is_spatial ex: `_check_is_spatial` and only resolve `if self.is_spacial == None`, in other words cache the calculated response. You also need to reset self.is_spatial to none in `Validator.reset()`